python/dolfinx/wrappers/mesh.cpp

```diff
@@ -98,8 +98,10 @@ void mesh(py::module& m)
       "cell_normals",
       [](const dolfinx::mesh::Mesh& mesh, int dim,
          const py::array_t<std::int32_t, py::array::c_style>& entities) {
-        return dolfinx::mesh::cell_normals(
+        auto normals = dolfinx::mesh::cell_normals(
             mesh, dim, tcb::span(entities.data(), entities.size()));
+        return py::array_t<double>(normals.shape, normals.strides(),
```

Comment:
If we add something like the `as_pyarray` function but for `common::array2d` we can avoid copying the data.